### ISSUE TITLE
Scoring

### ISSUE THREAD

**Issue by SigbjornR** [w3c/silver#281]
The Norwegian Digitalisation Agency (Digdir) is positive to WCAG 3.0’s approach to scoring and evaluation of conformance through different levels of conformance.

From an agency’s point of view, it is important to define what is good enough to constitute conformance. By splitting up the conformance for many of the outcomes in different levels, the result is more nuanced than just conformance/non-conformance. This is pedagogically wise, since it gives a clear indication on how close they are to reaching an acceptable level (bronze).

Norway has had legislation build on WCAG for several years. EU’s introduction of the Web Accessibility Directive (WAD) and the EU Accessibility Directive (EAA) will probably make WCAG more relevant for authorities in many other countries as well. We assume that the implementation of the two directives will ignite a call for harmonisation. WCAG 3.0 will hopefully help authorities, owners of ICT solutions and users to better understand and interpret the requirements of WCAG and directives and standards referring to WCAG. We believe the introduction of defined levels of conformance is a great start towards a unified approach.

Hence, Digdir welcomes this new development regarding WCAG and encourage the Working Group (WG) to go forward with this scoring model.

However, we hope the WG will clarify whether the different thresholds or other classifications are meant to indicate that some outcomes are more important than others. Such indications, if there are any, might be valuable for authorities when evaluating exceptions due to a disproportionate burden.

**Comment by jspellman** [comment]
Thank you for your helpful comments. The use case you identify in the last paragraph is not one that we had considered, and I am referring it to the group that are working on use cases to test the conformance model.

**Comment by sajkaj** [comment]
Thank you for this comment. Our [initial discussion is logged here](https://www.w3.org/2021/02/25-silver-conf-minutes.html#t02). We learned we need to understand your comment better. Could you perhaps say a bit more? Perhaps
provide us examples that illustrate your comment? And perhaps explain how and why this is important to you?

**Comment by SigbjornR** [comment]
This comment is closely connected to our role as a national authority for universal design of ICT and our use of WCAG in different legislation, e.g. national accessibility regulations in Norway (now WCAG 2.0) and EU’s Web Accessibility Directive (now EN 301 549/WCAG 2.1).

Both Norwegian law and WAD opens for exceptions if the requirements impose a disproportionate burden. The assessment on what imposes a disproportionate burden relies on many factors, such as cost, how much the solution is used, how essential the solution is the users etc., but also how essential the specific requirement is for users.
The importance of different requirements may change from solution to solution (both between websites and apps, but also between different websites and apps), but are there any requirements which the WG in general define as more essential than others?

Should for example the threshold for ascertaining a disproportionate burden be higher for e.g. a keyboard trap (WCAG 2.1. SC 2.1.2) than lack of text alternative to non-text content (WCAG 2.1 SC 1.1.1)?

**Comment by SigbjornR** [comment]
We would like to supplement the previous comments on scoring, since we have been made aware that they can be misinterpreted to support a more subjective approach. 

We welcome a more nuanced scoring than today’s, where a page is either fully conform or fail. By including different levels of partly conformance, WCAG will become more pedagogically towards the owners of websites, apps etc. by acknowledging what has been done and more clearly showing how much is left before the page meets the requirements. 

It is however important that the minimum level of conformance is communicated clearly in WCAG to avoid different interpretations. Especially considering the legal status WCAG 2.1 is given in WAD through the harmonisation of EN 301 549. 

An objective approach will not only make requirements more transparent, but also WCAG more relevant for defining legal requirements, such as EU legislation. Flexibility when it comes to how to interpret (and meet) the requirements is very challenging when the requirements apply across borders. The more objective WCAG requirements are, the more likely it is that the interpretation is harmonised, or at least very similar, in different countries with the same legal obligations. This is important for international harmonisation, and especially considering principles like mutual recognition within the EU’s single market.

Therefore, WCAG 3.0 should aim on becoming as objective as possible when defining the requirements. We welcome the idea of more requirements covering cognitive disabilities, but from an authority point of view, we strongly recommend the working group to aim for objective requirements. By doing so, the approach across borders will become both more harmonised and transparent, and WCAG will be more relevant for legally defining requirements.

**Comment by fstrr** [comment]
Proposed response:

This content has been updated with the release of the latest draft of WCAG 3, which can be found here:  [W3C Accessibility Guidelines \(WCAG\) 3.0](https://www.w3.org/TR/wcag-3.0/).  We believe that these issues are no longer relevant based on the changes made to the latest working draft.  Should you feel that these issues remain in the current draft, please log a new issue here:  [New Issue · w3c/wcag3](https://github.com/w3c/wcag3/issues/new).
